src/components/BottomNavigation/styles.scss

```diff
@@ -12,6 +12,7 @@
   z-index: 3000;
   width: 100%;
   max-width: 960px;
+  padding-left: 3px;
```

Comment:
Why more padding? 🤔

Reply:
The outline for the menu item on the left would most of the time get cut off at that end, so I thought a tiny padding might help without upsetting the rest too much.

Reply:
Oh ok! 👍